doc/install/python.rst

```diff
@@ -142,8 +151,8 @@ with MPI support would add the following to its requirements:
     arbor >= 0.3 --install-option='--gpu=cuda' \
```

Comment:
Not 0.4?

Reply:
It is an example, so it does not matter. I however am not sure if we need the section at all. Has anyone needed this instruction so far?